src/realm/CMakeLists.txt

```diff
@@ -0,0 +1,327 @@
+set(REALM_SOURCES
+    alloc.cpp
+    alloc_slab.cpp
+    array.cpp
+    array_binary.cpp
+    array_blob.cpp
+    array_blobs_big.cpp
+    array_integer.cpp
+    array_string.cpp
+    array_string_long.cpp
+    bptree.cpp
+    column.cpp
+    column_backlink.cpp
+    column_binary.cpp
+    column_link.cpp
+    column_link_base.cpp
+    column_linklist.cpp
+    column_mixed.cpp
+    column_string.cpp
+    column_string_enum.cpp
+    column_table.cpp
+    column_timestamp.cpp
+    descriptor.cpp
+    disable_sync_to_disk.cpp
+    exceptions.cpp
+    group.cpp
+    group_shared.cpp
+    group_writer.cpp
+    history.cpp
+    impl/output_stream.cpp
+    impl/simulated_failure.cpp
+    impl/transact_log.cpp
+    index_string.cpp
+    lang_bind_helper.cpp
+    link_view.cpp
+    query.cpp
+    query_engine.cpp
+    query_expression.cpp
+    replication.cpp
+    row.cpp
+    spec.cpp
+    string_data.cpp
+    table.cpp
+    table_view.cpp
+    unicode.cpp
+    util/basic_system_errors.cpp
+    util/encrypted_file_mapping.cpp
+    util/file.cpp
+    util/file_mapper.cpp
+    # util/interprocess_condvar.cpp
+    # util/interprocess_mutex.cpp
```

Comment:
No they don't. They are added a few lines below for non-Windows platforms. The comments are there to make it easier to spot when Windows will add multi-process support.